static/styles/left_sidebar.css

```diff
@@ -67,6 +67,54 @@ li.show-more-topics {
     }
 }
 
+li#show-more-private-conversations {
+    cursor: pointer;
+    padding-top: 1px;
+    padding-bottom: 1px;
+
+    a {
+        font-size: 12px;
+        padding-left: 32px;
+        color: inherit;
+        text-decoration: none;
+    }
+
+    .unread_count {
+        margin-right: 42px;
```

Comment:
This seems wrong; the unread count is not aligned with the rest at all.

![image](https://user-images.githubusercontent.com/2746074/184265654-3e1a4ecb-a872-480a-aa87-b4ddeddb1149.png)

Reply:
More generally, this CSS has a huge number of magic numbers like this 42px that seem unlikely to be correct.

Reply:
I fixed this CSS error and pushed it back;

I agree that this CSS has some `magic numbers` but I'm not sure how should I be able to make them look good to you ?, as all those are variables which I've looked at in local dev and then put them here.
maybe there's some way you want me to follow here which makes it easier for you to review and approve ?

Reply:
Generally, the goal is to write CSS that looks correct without a lot of hand-tuned numbers, and if you do need any, to figure out how they related to each other and use CSS variables to define that, like we have at the top of `left_sidebar.css`:
```
/* The width of the empty space in the sidebar to separate                            
   content from the edge of the window */                                             
$far_left_gutter_size: 10px;                                                          
/* This represents the space in the sidebar reserved for symbols like                 
   the #; labels like the the stream name go to the right of this. */                 
$left_col_size: 19px;                                                                 
/* The full topic indentation includes 4px of indent in addition to                   
   the above (and another 5px of padding not measured here) */                        
$topic_indent: calc($far_left_gutter_size + $left_col_size + 4px);                    
$topic_resolve_width: 13px;                                                           
```